src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
does it work without the upper tag? please share a screenshot

Reply:
Yup, it does work
https://i.imgur.com/1sMF633.png

Will add it anyway)